test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -6,6 +6,7 @@
 
 #include "ucx_wrapper.h"
 
+#include <ucs/debug/log_def.h>
```

Comment:
why needed?

Reply:
added during debugging, now it is not needed
removed